test/JSONFileReaderWrongColNumberTestFile.json

```diff
@@ -3,6 +3,7 @@
   "colNumber": 1,
   "rows": [
     [1, 1],
+    [1],
```

Comment:
This change is not associated with this task. It shouldn't be there.

Reply:
same as below